README.md

```diff
@@ -0,0 +1,12 @@
+# Zattoo PVR addon for Kodi
+
+This is a [Kodi] (http://kodi.tv) PVR addon for streaming live TV from [zattoo](https://zattoo.com).
+
+## Build instructions
+
+1. `git clone -b zattoo https://github.com/rbuehlma/xbmc`
+2. `git clone -b zattoo https://github.com/rbuehlma/pvr.zattoo.git`
+3. `cd pvr.zattoo && mkdir build && cd build`
+4. `cmake -DADDONS_TO_BUILD=pvr.zattoo -DADDON_SRC_PREFIX=../.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=../../xbmc/addons -DPACKAGE_ZIP=1 ../../xbmc/project/cmake/addons`
+5. `make`
+6. `sudo zip -r pvr.zattoo.0.1.25.zip ../../xbmc/addons/pvr.zattoo && mv pvr.zattoo.0.1.25.zip ~`
```

Comment:
Use `make package-pvr.zattoo` instead of 5. and 6. as that will create a zip as output.